tests/CLI/modules/securitygroup_tests.py

```diff
@@ -148,6 +158,8 @@ def test_securitygroup_rule_edit(self):
                                 args=([{'id': '520',
                                         'direction': 'ingress'}],))
 
+        self.assertEqual([{'requestId': 'editRules'}], json.loads(result.output))
```

Comment:
I like that you use a real data structure (not a string) for the 'actual', and hence do a json.loads on the 'expected'. Helps to guard against typos and invalid data, and keeps the test working even if the output changes in semantically transparent ways like whitespace.